packages/s2-core/__tests__/unit/utils/export/method-spec.ts

```diff
@@ -11,21 +8,53 @@ describe('method test', () => {
     expect(keyEqualTo('a', '')).toBeFalsy();
     expect(keyEqualTo('A', 'a')).toBeTruthy();
   });
+});
+
+describe('escapeField', () => {
+  it('should return the same value for non-string types', () => {
```

Comment:
建议用 each，另外这两个单测感觉可以合并吧？

```suggestion
  it.each([42, null, undefined,'hello', '123', 'test']])('should return the same value for non-string types %s', (input) => {
  expect(escapeField(input)).toBe(input);
  }
```